Errors during installation

Hi, I was attempting a vanilla install of your library and observed several problems which are described below

## Environment:
Ubuntu 22.04 WSL
Python: 3.9.12
Cython: 0.29.28

## Commands:
```bash
git clone https://github.com/ehw-fit/evoapproxlib.git
git checkout v2022
cd evapproxlib
python3 make_cython.py
cd cython
python3 setup.py build_ext
python3 setup.py install --user
```
## Expected Behavior:
Successful Install

## Actual Behavior:
There are several problems occuring 
### 1. Warnings in all Cython files
```
/home/.../miniconda3/lib/python3.9/site-packages/Cython/Compiler/Main.py:369: FutureWarning:
Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release!
File: /home/.../approx_libs/evoapproxlib/cython/adders_16_unsigned_add16u_0UV.pyx  
```

### 2. Compilation errors in some Cython files
Several Cython files produce errors when passed to the Cython compiler.
```
cythoning multipliers_8x7_unsigned_mul8x7u_31P.pyx to multipliers_8x7_unsigned_mul8x7u_31P.c                                                                                                                                                                                                                                                                                                                                                                                              Error compiling Cython file:                                                                                                                                                                                                                 ------------------------------------------------------------                                                                                                                                                                                 ...                                                                                                                                                                                                                                          # from multiplers/8x7_unsigned/pareto_pwr_wce/mul8x7u_31P.c
from libc.stdint cimport *
cdef extern:
uint64_t c_mul8x7u_31P "mul8x7u_31P" (const uint64_t A /* 8-bit unsigned operand */, const uint64_t B /* 7-bit unsigned operand */)
------------------------------------------------------------
multipliers_8x7_unsigned_mul8x7u_31P.pyx:6:59: Expected ')', found '/'
```
The problem is that the parser does not correctly handle the inline comments that state the respective datatypes

### 3. Error during C compilation
Due to the incorrect parsing, the build step fails for several submodules.
```
building 'evoapproxlib.add8s_7EL' extension 
wrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.8 -c ../adders/8_signed/pareto_pwr_mae/add8s_7EL.c -o build/temp.linux-x86_64-3.8/../adders/8_signed/pareto_pwr_mae/add8s_7EL.o
x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.8 -c adders_8_signed_add8s_7EL.c -o build/temp.linux-x86_64-3.8/adders_8_signed_add8s_7EL.o
adders_8_signed_add8s_7EL.c:1:2: error: #error Do not use this file, it is the result of a failed Cython compilation.
```
## Solutions
Both issues are easy fixes.
1. The warning can be fixed by setting the `language_level` directive in each generated .pyx file: https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#globally. This can be added in the [file generation](https://github.com/ehw-fit/evoapproxlib/blob/5c6185e1e19dd7553e1c6c0946983312b56724e4/make_cython.py#L105-L107)
2. I have used this additional RegEx: `re.sub(r'\/\*[\w\d -]*\*\/', '', m)` in the [generation of the function header](https://github.com/ehw-fit/evoapproxlib/blob/d321f7f68c81141a7eed7556e1614b86d4ba629a/make_cython.py#L42-L43) to remove the inline comments altogether 

With these changes, compilation works without issues for me.
If you want to upstream these fixes, I can open a PR.

--- a/.github/workflows/test_install.yaml
+++ b/.github/workflows/test_install.yaml
@@ -1,0 +1,21 @@
+name: Test Install
+on: [push]
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - name: Install Python
+              uses: actions/setup-python@v4
+              with:
+                  python-version: '3.9'
+            - name: Install Cython
+              run: pip install cython
+            - name: Create Package
+              run: python3 make_cython.py
+            - name: Build and Install
+              run: |
+                  cd cython
+                  python3 setup.py build_ext
+                  python3 setup.py install

--- a/make_cython.py
+++ b/make_cython.py
@@ -40,7 +40,8 @@ def find_func(name, c_file):
     name = re.escape(name)
     m = re.search(r'\n.+\s' + name + r'\s*\(.+\)\s*\{', contents)
     if m:
-        return re.sub(r'\)\s*\{$', ')', m.group(0)).strip()
+        m = re.sub(r'\)\s*\{$', ')', m.group(0)).strip()
+        return re.sub(r'\/\*[\w\d -]*\*\/', '', m)
 
 def alias_func(func):
     return re.sub(r'^([a-zA-Z0-9_]+)\s+([a-zA-Z0-9_]+)', r'\1 c_\2 "\2" ', func)
@@ -103,6 +104,7 @@ def main():
 
             with open(pyxfile, 'w') as pyx:
                 pyx.write(f'# from {path.relative_to(root)}\n')
+                pyx.write('# cython: language_level=3\n')
                 pyx.write('from libc.stdint cimport *\n')
                 pyx.write('\n')
                 pyx.write('cdef extern:\n')

